Automattic/woocommerce-payments
# Fix checkout page auto scroll issue

#### Changes proposed in this Pull Request
The changes made in the platform checkout auto redirection PR https://github.com/Automattic/woocommerce-payments/pull/4328 introduced a scroll issues. The checkout page scrolls down a little when the page is loaded. This PR fixes that and prevents the unwanted auto scroll.

#### Steps to create the issue
- Add some products to the cart and go to the checkout page.
- The checkout page automatically scrolls down a bit. 

#### Testing instructions
- Add some products to the cart and go to the checkout page.
- The checkout page should not scroll down
